pkgs/development/python-modules/zerobin/default.nix

```diff
@@ -1,13 +1,13 @@
 { lib
```

Comment:
So is this file - `python-modules/zerobin/default.nix` being used anywhere?

Reply:
Oh, right, that's the old one, I've removed it, thanks.